### DISCUSSION TITLE
Users can read all settings, is this by design?

### DISCUSSION THREAD

**Discussion by witsec** [directus/directus#2880] [Archive]
I really like how easy it is to set up roles and permissions, but I noticed one thing today (on Directus 8.8.1). I created a new role and gave the role permissions to access a certain collection. On one browser I was logged in as admin, on another as a user that's part of the role.

At some point I copied the URL of the admin window to the user window (basically this: `/public/admin/#/Directus/settings`). Then I noticed the user was able to see all settings, all collections (of course they could only really see the collection I gave permission on, but still), etc.  So I changed the read permissions on "settings" from the default "all" to "only me", but that caused the user's window to kinda break. I had to change that back to "all" so the user could work again.

The user obviously isn't able to change anything, but I don't like the idea of them being able to snoop around. Of course you have to kinda know the URL but still. Or is this by design?

Thanks

**Comment by benhaynes** [comment]
Thanks for reporting this! This is not supposed to be like this, and has been resolved in version 9.
